lib/osf-components/addon/components/search-page/styles.scss

```diff
@@ -1,206 +1,328 @@
+            display: inline-flex;
+            white-space: nowrap;
+            margin-right: 10px;
+
+            &.active,
+            &:hover {
+                background-color: $color-bg-gray;
+            }
         }
-    }
 
-    span {
-        padding: 0 1rem 1rem;
+        a {
+            padding-bottom: 10px;
+        }
     }
 
-    > button {
-        width: 100%;
-        margin-top: 0.5rem;
+    .help-button {
+        margin-left: 1.2rem;
     }
 }
 
-.heading-label {
-    padding: 40px;
-}
-
-.search-input-wrapper {
-    white-space: nowrap;
-}
-
-.search-input {
-    color: $color-text-black;
-    padding: 9px 5px;
-    font-size: 1.5em;
-    max-width: 700px;
-    min-width: 250px;
-    width: 50vw;
-}
-
-.help-button,
-.search-button {
-    position: relative;
-    right: 3px;
-    bottom: 3px;
-}
-
-.topbar {
-    margin-left: 20px;
-    display: flex;
-    justify-content: space-between;
-    border-bottom: 1px solid $color-text-black;
-}
-
-.object-type-nav {
-    ul {
-        padding-left: 0;
-        margin-bottom: 0;
-        list-style: none;
-    }
-
-    li {
-        display: inline-flex;
-        margin-right: 10px;
-    }
+.search-main {
+    display: grid;
+    justify-content: center;
 }
 
 .search-help-tutorial {
     background-color: $osf-dark-blue-navbar;
     color: $color-text-white;
     width: 400px;
-    max-width: 100%;
-    z-index: 100;
-
-    // prevent target element styles from dictating popover text-styles
-    white-space: wrap;
-    text-align: initial;
+    z-index: 10;
+    
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    margin-left: -0.2rem;
 
-    .enumeration {
-        float: left;
+    h2 {
+        font-weight: 400;
+        height: 22px;
+        margin-top: 0.5rem;
+        margin-left: 0.2rem;
     }
-    
-    .help-button-wrappers {
-        float: right;
+
+    p {
+        white-space: normal;
+        margin: 25px 5px 20px;
     }
 
     .skip-button {
         color: $color-text-white;
+        overflow: visible;
+    }
+
+    .pagination {
+        display: flex;
+        justify-content: space-between;
+        align-items: baseline;
     }
-}
 
+    .help-button-wrapper {
+        display: flex;
+        justify-content: flex-end;
+
+        button:first-of-type {
+            margin-right: 20px;
+        }
+    }
+}
 
 .object-type-filter-link {
-    padding: 10px 15px;
     font-size: 1.3em;
+    padding: 18px 15px;
 
     &:global(.active),
     &:hover {
-        text-decoration: none;
-        background-color: $color-bg-gray;
+        background-color: $color-bg-white;
+        border-bottom: 1px solid $color-text-black;
         color: $color-text-black;
-        border-bottom: 2px solid $color-text-black;
+        text-decoration: none;
     }
 }
 
 .sort-dropdown {
-    width: 170px;
-    margin-top: 10px;
+    background-color: $color-bg-gray-blue-light;
+    display: flex;
+    flex-direction: column-reverse;
```

Comment:
Almost, this actually affects the alignment of the text contained by the inner div of the sort dropdown . `align-items: flex-end; `also works. In this scenario, since there are not multiple items to reverse I can use the one for positioning not order. Updating now.

Reply:
Almost, this actually affects the alignment of the text contained by the inner div of the sort dropdown. `align-items: flex-end; ` also works. In this scenario, since there are not multiple items to reverse I can use the one for positioning not order. Updating now.